packages/openai/src/responses/openai-responses-language-model.ts

```diff
@@ -104,6 +104,14 @@ export class OpenAIResponsesLanguageModel implements LanguageModelV2 {
 
     const strictJsonSchema = openaiOptions?.strictJsonSchema ?? false;
 
+    // Determine which includes are supported by the OpenAI API and which are SDK-only.
+    const requestedIncludes = openaiOptions?.include ?? undefined;
+    const apiSupportedIncludes = (requestedIncludes ?? []).filter(include =>
+      ['reasoning.encrypted_content', 'file_search_call.results'].includes(
+        include,
+      ),
+    );
```

Comment:
would be nice to extract a generic intersection function for 2 arrays

Reply:
removed for now, no more partial images support